data/src/test/java/org/apache/iceberg/io/TestRollingFileWriters.java

```diff
@@ -169,9 +170,9 @@ public void testRollingEqualityDeleteWriterSplitDeletes() throws IOException {
     writer.close();
 
     DeleteWriteResult result = writer.result();
-    Assert.assertEquals(4, result.deleteFiles().size());
-    Assert.assertEquals(0, result.referencedDataFiles().size());
-    Assert.assertFalse(result.referencesDataFiles());
+    assertThat(writer.result().deleteFiles()).hasSize(4);
```

Comment:
you may want to remove the `result` variable if you're using it directly here

Reply:
Oh sorry, I will update this part.